common/database/database_dump_service.cpp

```diff
@@ -93,19 +97,30 @@ bool DatabaseDumpService::IsTarAvailable()
  * Windows
  * @return bool
  */
-bool DatabaseDumpService::Is7ZipAvailable()
+bool DatabaseDumpService::Is7ZipAvailable(std::string &cmd)
 {
-	std::string version_output = execute("7z -help");
+	static const std::string command_list[] = { "7z --help", "\"c:/Program Files/7-Zip/7z\" --help" };
+
+	for (auto &command : command_list) {
+
+		if (execute(command).find("7-Zip") != std::string::npos) {
+
+			cmd = command;
+			cmd.erase(cmd.find(" --help"));
 
-	return version_output.find("7-Zip") != std::string::npos;
+			return true;
+		}
+	}
+
+	return false;
 }
 
 /**
  * @return
  */
 bool DatabaseDumpService::HasCompressionBinary()
 {
-	return IsTarAvailable() || Is7ZipAvailable();
+	return IsTarAvailable() || Is7ZipAvailable(std::string());
```

Comment:
Is there a reason we're passing an empty string here? Can we just as well remove it ?